SPECS/dbus-python/dbus-python.spec

```diff
@@ -0,0 +1,324 @@
+%global python3_pkgversion 3.7
+%global __python3 \/usr\/bin\/python3
+
+Summary:        D-Bus Python Bindings
+Name:           dbus-python
+Version:        1.2.16
+Release:        6%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Applications/System
+URL:            http://www.freedesktop.org/wiki/Software/DBusBindings/
+Source0:        http://dbus.freedesktop.org/releases/dbus-python/%{name}-%{version}.tar.gz
+
+Patch0: 0001-Move-python-modules-to-architecture-specific-directo.patch
+
+BuildRequires: gnupg2
+BuildRequires: glib-devel>=2.42.2
+BuildRequires: dbus>=1.8.6
+BuildRequires: dbus-devel>=1.8.6
+%if %{with_check}
+BuildRequires: python3-pip
+%endif
+
+Requires:      zlib
+Requires:      openssl
+Requires:      c-ares
+
+%description
+D-Bus python bindings for use with python programs
+
+%package -n python3-dbus
+Summary: D-Bus bindings for python3
+%{?python_provide:%python_provide python%{python3_pkgversion}-dbus}
+BuildRequires: python3-devel
+BuildRequires: make
+ 
+%description -n python3-dbus
+%{summary}.
+
+%package devel
+Summary: Libraries and headers for dbus-python
+ 
+%description devel
+Headers and static libraries for hooking up custom mainloops to the dbus python
+bindings.
+
+%prep
+%autosetup -p1
+
+%build
+%set_build_flags
+export PYTHON=python3
+%configure PYTHON="%{__python3}"
+%make_build
+
+%install
+%make_install
+
+%check
+pip3 install xml
+make check -k || (cat test-suite.log && false)
+
+%files -n python3-dbus
+%{_libdir}/python3.7/site-packages/*.so
+%{_libdir}/python3.7/site-packages/dbus/
+
+%files devel
+%{_includedir}/dbus-1.0/dbus/dbus-python.h
+%{_libdir}/pkgconfig/dbus-python.pc
+%exclude %{_libdir}/python3.7/site-packages/*.la
+
+%changelog
+* Fri Jul 30 2021 Neha Agarwal <nehaagarwal@microsoft.com> - 1.2.16-6
+- Initial CBL-Mariner import from Fedora 34 (license: MIT)
+- Removed GPG signature, pkgconfig for BuildRequires
```

Comment:
Double check that the dbus-python license is and add a License Verified statement in the changelog to confirm.

Reply:
Done